workflow-templates/cfgov-python-satellite-ci.yml

```diff
@@ -22,6 +27,8 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
This change was necessary due to a change in checkout@v2 that defaults to shallow cloning with no tags. I have opened a [feature request issue](https://github.com/actions/checkout/issues/338) to ask for the option to shallow clone but also pull down tags.